protected/views/patient/crud/_form.php

```diff
@@ -417,4 +406,20 @@
       $this->endWidget('zii.widgets.jui.CJuiDialog');
      ?>
 
-</div><!-- form -->
\ No newline at end of file
+</div><!-- form -->
+
+<script type="text/javascript">
+  function findDuplicates() {
+    if ($("#fname").val() && $('#surname').val() && $('#patient_dob').val()) {
+      $.ajax({
+          url: "<?php echo Yii::app()->controller->createUrl('patient/findDuplicates'); ?>" + "?firstName=" + $('#fname').val() + "&surname=" + $('#surname').val() + "&dob=" + $('#patient_dob').val(),
```

Comment:
Instead of concatenating the firstname/lastname/dob, you can instead use the data: argument for .ajax()